distributor-node/src/command-base/default.ts

```diff
@@ -98,6 +100,21 @@ export default abstract class DefaultCommandBase extends Command {
     }
   }
 
+  async datePrompt(question: DistinctQuestion): Promise<Date> {
+    const { result } = await inquirer.prompt([
+      {
+        ...question,
+        type: 'datepicker',
+        name: 'result',
+        clearable: true,
+        default: new Date('2017-09-28 17:36:05').toISOString(),
```

Comment:
Tried the date picker, works really nicely in the terminal!

Perhaps a more reasonable default would be the current date `new Date().toISOString()`, so less fiddling with inital year and month for the user.

Reply:
Done